src/time-lru-set.test.js

```diff
@@ -0,0 +1,86 @@
+//
+// Copyright 2019 DxOS.
+//
+
+import TimeLRUSet from './time-lru-set';
+
+const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+describe('time lru set', () => {
+  const timeSet = new TimeLRUSet({
+    maxAge: 1 * 1000,
+    maxSize: 2
+  });
+
+  test('default timeSet', () => {
+    const timeSet = new TimeLRUSet();
+    expect(timeSet._maxAge).toBe(10 * 1000);
+    expect(timeSet._maxSize).toBe(100);
+  });
+
+  test('time live', async () => {
+    timeSet.add('foo');
```

Comment:
Great tests, but perhaps in general for tests a standard pattern like:

lodash.times(10, i => timeSet.add(`item-${i}`)

Instead of foo/bar, since we may often want to test limits.